### PR TITLE
HW05 Poshtarenko Andrei

### PR DESCRIPTION
**Pull request by EKCKABATOP54**

### CODE REVIEW

File: expr/src/Expr.hs [hunk 1]
```diff
@@ -1,22 +1,80 @@
-module Expr where 
+module Expr where
```
Comment (kajigor) [inline expr/src/Expr.hs:1]: It's better to split the functionality into multiple modules, each of which does its own thing

**Review by kajigor — CHANGES_REQUESTED** [review]

**Comment by kajigor** [comment]
Have you made tasty work in the end?

File: expr/src/Expr.hs (hunk 1 above)
Comment (EKCKABATOP54) [reply expr/src/Expr.hs:1]: The implementation of Error depends on Expr for more detailed error reporting. Therefore, when dividing into modules, a cyclic dependence is obtained

**Comment by EKCKABATOP54** [comment]
> Have you made tasty work in the end?
Yes. For some reason an error occurred on Windows, but on Linux everything is ok

I have one more question. The PR of the fourth homework closed with the comment “ok”, which means, as I understand it, everything is ok. But the score for task 4 remained the same as before the last fix. So I would like to ask if the scores in the table are correct?

**Comment by kajigor** [comment]
HW04: fixed

File: expr/src/Expr.hs (hunk 1 above)
Comment (EKCKABATOP54) [reply expr/src/Expr.hs:1]: Fixed

**Comment by kajigor** [comment]
ok

**Pull request closed without merging**